src/lib/pmd/PmdCatalogWrapper.ts

```diff
@@ -117,6 +117,9 @@ export class PmdCatalogWrapper extends PmdSupport {
 		// Now, we'll want to add the default PMD JARs for any activated languages.
 		const supportedLanguages = await PmdLanguageManager.getSupportedLanguages();
 		supportedLanguages.forEach((language) => {
+			if (LANGUAGES.JAVASCRIPT === language) {
```

Comment:
This seemed like the most centralized place to throw the error. The code isn't checking the glob patterns. Even if the glob patterns are present, the files aren't analyzed because the PMD javascript rules aren't included.

Reply:
Should this check be within PmdLanguageManager.getSupportedLanguages()? The method name sounds relevant to the check. Also, there's an existing foreach loop to look at each language string before sending the list here.

Reply:
good idea, i'll move it